templates/index.html

```diff
@@ -13,8 +13,8 @@
     <section class="full-width-section">
         <div class="section-content">
             <div class="header-with-image">
-                <img src="/static/img/corn.png">
-                <h2>Test Setup Corn</h2>
+                <img id="setup-icon">
```

Comment:
I believe the setups can instead be passed as arguments for the `render_template` function. Since they don't change during a page render, javascript should not be required

Reply:
No longer request using the javascript, but still keeping the API endpoint, since it is necessary anyway